index.html

```diff
@@ -0,0 +1,1342 @@
+                            </path>
+                        </g>
+                    </svg>
+                </div>
+                <div class="login-section d-flex justify-content-center align-items-center align-self-center align-items-lg-start align-self-lg-end">
+                    <button class="d-none btn login-btn d-md-block ">LOGIN</button>
+                    <i class="fa fa-user d-md-none fa-lg" aria-hidden="true"></i>
+
+                    <button class="btn btn-warning sub-btn rounded-pill font-wight-bold text-nowrap mx-1"> SUBSCRIBE 
+                        <i class="fa fa-chevron-right" aria-hidden="true"></i></button>
+                    <i class="menu fa fa-bars fa-lg d-lg-none" aria-hidden="true"></i>
+                </div>
+            </div>
+
+            <div class="d-none d-lg-block news-category  position-relative">
+                <ul class="list-unstyled d-flex flex-wrap">
+                    <li class="news-cat-li"><a href="#">U.S</a></li>
+                    <li class="news-cat-li"><a href="#">World</a></li>
+                    <li class="news-cat-li"><a href="#">Business</a></li>
+                    <li class="news-cat-li"><a href="#">Tech & Science</a></li>
+                    <li class="news-cat-li"><a href="#">Culture</a></li>
+                    <li class="news-cat-li"><a href="#">Newsgeek</a></li>
+                    <li class="news-cat-li"><a href="#">Sports</a></li>
+                    <li class="news-cat-li"><a href="#">Health</a></li>
+                    <li class="news-cat-li"><a href="#">The Debate</a></li>
+                    <li class="news-cat-li"><a href="#">Experts</a></li>
+                    <li class="news-cat-li"><a href="#">Vantage</a></li>
+                    <li class="search-li"><a href="#">Search <i class="fa fa-search" aria-hidden="true"></i></a>
+                    </li>
+
+                </ul>
+            </div>
+            <div class="d-none d-lg-block trending-news">
+                <ul class="list-unstyled d-flex flex-wrap">
+                    <li><a href="#" class="trending-news-li">TRENDING</a></li>
+                    <li><a href="#" class="trending-news-li">JOE BIDEN</a></li>
+                    <li><a href="#" class="trending-news-li">DONALD TRUMP</a></li>
+                    <li><a href="#" class="trending-news-li">INUGURATIONS</a></li>
+                    <li><a href="#" class="trending-news-li">CORONAVIRUS</a></li>
+                    <li><a href="#" class="trending-news-li">STIMULUS PACKAGE</a></li>
+                    <li><a href="#" class="trending-news-li">VACINE</a></li>
+                    <li><a href="#" class="trending-news-li">CAPITOL RIOTS</a></li>
+                </ul>
+            </div>
+        </nav>
+         
+    </header>
+
+    <div class="main row mx-1">
```

Comment:
- [ ] Why not use the more semantic `main` tag instead of a div here, since you are already using the `.main` class.